### PR TITLE
OC: ua trusty enable esm

### PR DESCRIPTION
**Pull request by panlinux**
Support enable ESM on trusty

### CODE REVIEW

File: tests/test_esm.py [hunk 1]
```diff
@@ -10,19 +10,33 @@ class ESMTest(UbuntuAdvantageTest):
...
+            '# deb-src https://user:pass@esm.ubuntu.com/ubuntu '
+            'trusty-updates main\n')
+        for series in ['precise', 'trusty']:
+            self.SERIES = series
+            process = self.script('enable-esm', 'user:pass')
+            self.assertEqual(0, process.returncode)
+            self.assertIn('Ubuntu ESM repository enabled', process.stdout)
+            self.assertEqual(expected_repo_list[series],
+                             self.repo_list.read_text())
+            keyring_file = self.trusted_gpg_dir / 'ubuntu-esm-keyring.gpg'
```
Comment (powersj) [inline tests/test_esm.py]: os.path.join(self.trusted_gpg_dir, 'ubuntu-esm-keyring.gpg')

of course we will need something different for trusty

Comment (panlinux) [reply tests/test_esm.py]: I have another branch to add the unattended-upgrades config, it's https://github.com/panlinux/ubuntu-advantage-client/tree/oc-unattended-upgrades-pockets

I wanted to land things piece by piece.

About the gpg key, do we have it already, or should I just make one up and replace it when the time comes? I also wanted to do a separate branch for that.

Comment (powersj) [reply tests/test_esm.py]: If you fix the keyring_file creation I'll +1 this.

Let's ask about the GPG key in standup tomorrow.

I'll take al ook at the UU config next.

Comment (panlinux) [reply tests/test_esm.py]: Ah, I see what you mean now about the keyring. Looks like I will have to handle that in this branch.

**Comment by panlinux** [comment]
I'm installing both keyrings in the case of an ESM supported distro. Should I split that up? On trusty, only install the trusty keyring? I haven't fully considered the precise to trusty release upgrade scenario.

**Pull request merged**